images/Dockerfile

```diff
@@ -53,8 +53,8 @@ ARG NODEJS_VER=20
 #
 ARG LOTMAN_SRC_BUILD=false
 ARG LOTMAN_VER=0.0.4
-ARG XRDCL_PELICAN_SRC_BUILD=false
-ARG XRDCL_PELICAN_VER=1.2.3
+ARG XRDCL_PELICAN_SRC_BUILD=true
+ARG XRDCL_PELICAN_VER=1.3.1
```

Comment:
This version should also be updated in `github_scripts/osx_install.sh`. When you do that, can you add a note/warning comment above this section so developers know there's a dependency update required in both places?